psd-web/app/forms/supporting_information_form.rb

```diff
@@ -0,0 +1,17 @@
+class SupportingInformationForm
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+
+  AVAILABLE_TYPES = {
+    comment: "Comment or case note",
```

Comment:
```suggestion
    comment: "Comment",
```

Sorry, 'case note' was a speculative thing for the future.